src/Tribe/Modules/Single_Event_Template.php

```diff
@@ -135,6 +141,15 @@ public function tec_ea_single_event_template( array $event, \Tribe__Events__Aggr
 			);
 		}
 
+		// If there is a venue in the source, then change it in the template.
+		if ( isset( $event['EventVenueID'] ) ) {
```

Comment:
We can validate the Venue ID this way to ensure it's a valid ID
```suggestion
		if ( ! empty( $event['EventVenueID'] ) && intval( $event['EventVenueID'] ) > 0 ) {
```